### PR TITLE
Added automated docs

### PR DESCRIPTION
**Pull request by Sjoertjuh**
Soon to be #⁠1 on the Insights page 😉 

*Note: The TODO's cannot be fixed right now, I'll try to create an issue in the [vue-styleguidist](https://github.com/vue-styleguidist/vue-styleguidist) repo to add that feature*

Relevant issues:
- https://github.com/vue-styleguidist/vue-styleguidist/issues/1330
- https://github.com/vue-styleguidist/vue-styleguidist/issues/1285

### CODE REVIEW

File: src/components/Tooltip/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    Insert coin here
+</Tooltip>
+```
+
+## Imported Props
+
+| Prop name          | Description                                | Type                                                              | Values                                | Default                            |
+| ------------------ | ------------------------------------------ | ----------------------------------------------------------------- | ------------------------------------- | ---------------------------------- |
+| fallbackPlacements | The fallback placements of the component   | [TooltipFallbackPlacement](../../composables/useTooltipPlacement) | ['top'\|'bottom'\|'left'\|'right']    | ['top', 'right', 'bottom', 'left'] |
+| placement          | The default placement of the component     | [TooltipPlacement](../../composables/useTooltipPlacement)         | ['top'\|'bottom'\|'left'\|'right']    | 'top'                              |
```
Comment (TimDirks) [inline src/components/Tooltip/README.md:14]:
```suggestion
| placement          | The default placement of the component     | [TooltipPlacement](../../composables/useTooltipPlacement)         | ['top'\|'bottom'\|'left'\|'right'\|'auto']    | 'top'                              |
```

File: src/components/Tooltip/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## Events
+
+| Name          | Args | Description                                                                                             |
+| ------------- | ---- | ------------------------------------------------------------------------------------------------------- |
+| show          |      | This event fires immediately when the `show` instance method is called                                  |
+| shown         |      | This event is fired when the popover has been made visible to the user                                  |
+| hide          |      | This event is fired immediately when the `hide` instance method has been called                         |
+| hidden        |      | This event is fired when the popover has finished being hidden from the user                            |
+| hidePrevented |      | This event is fired after the show.bs.tooltip event when the tooltip template has been added to the DOM |
```
Comment (TimDirks) [inline src/components/Tooltip/README.md:27]:
```suggestion
| inserted |      | This event is fired after the show.bs.tooltip event when the tooltip template has been added to the DOM |
```

**Review by TimDirks — APPROVED** [review]

File: src/components/Tooltip/README.md (hunk 1 above)
Comment (Sjoertjuh) [reply src/components/Tooltip/README.md:14]: It exists in `placement` under `Options` https://getbootstrap.com/docs/5.2/components/tooltips/

Comment (TimDirks) [reply src/components/Tooltip/README.md:14]: I know, that's why I suggested to add it

Comment (Sjoertjuh) [reply src/components/Tooltip/README.md:14]: Ah, I added it somewhere else then. I know I added it somewhere... smh

**Review by thomasowow — CHANGES_REQUESTED** [review]
So how and when should we run this? Should we automatically run it? At least at something to readme.

The somposables should have examples.

**Review by thomasowow — APPROVED** [review]

**Pull request merged**